### PR TITLE
Add PowerAPI toolkit scripts and user guide

### PR DESCRIPTION
**Pull request by fmuyassarov**

### CODE REVIEW

File: powerapi/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+program.
+
+## Power API
+
+Power API is kind of an umbrella of multiple tools, targeting measurement of energy
+consumption from different sources. Regardless of which one you use, ensure that the host
+you are trying to measure is based on Intel's Sandy Bridge+ processor architecture.
+
+Under the hood Power API is relaying on Inte's Running Average Power Limit(RAPL) to
+get power meter
```
Comment (kashifest) [inline powerapi/README.md:17]: power reading?

File: powerapi/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Architecture
+
+![](images/powerapi.svg)
+
+There are a couple of actors involved in the process.
+ - Sensor - collects actual power consumptions from the host
+ - Formulas - converts raw data from sensor into human readable format
+ - Databases - store sensor and formula data
```
Comment (kashifest) [inline powerapi/README.md:26]: stores

**Review by kashifest — DISMISSED** [review]
Small nits

File: powerapi/sensor.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+-v $(pwd)/config_file_sensor.json:/config_file_sensor.json \
+powerapi/hwpc-sensor \
+--config-file /config_file_sensor.json
+
+
+# Connect to mongoDB
+# mongo 172.17.0.2:27017
+
+# start InfluxDB
+# docker run -d --name influx_rapl -p 8086:8086 influxdb:1.8
```
Comment (furkatgofurov7) [inline powerapi/sensor.sh:20]: nit: do we need those?

File: powerapi/README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    ```shell
+    $ docker run -d --name influx_rapl -p 8086:8086 influxdb:1.8
+    ```
+
+3. Start the sensor. We will be running the sensor as docker container but there are some other ways
+    also to run it. The sensor script takes config_file_sensor.json as configuration file.
+
+    ```shell 
+     $ git clone https://github.com/fmuyassarov/power.git
+     $ cd power
```
Comment (furkatgofurov7) [inline powerapi/README.md:70]: why do we need to clone, that script is already added here powerapi/sensor.sh, so:
```suggestion
     $ cd powerapi
```
should be enough

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: powerapi/sensor.sh (hunk 3 above)
Comment (fmuyassarov) [reply powerapi/sensor.sh:20]: I think yes, just to know how to connect to DB in case

**Comment by fmuyassarov** [comment]
@furkatgofurov7 @kashifest Thanks for your comments. PTAL once more.

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**